youtube_dl/extractor/pornorama.py

```diff
@@ -0,0 +1,50 @@
+from __future__ import unicode_literals
+
+import re
+
+from .common import InfoExtractor
+from ..compat import compat_urllib_parse_unquote
+from ..utils import (
+    clean_html,
+    ExtractorError
+)
+
+
+class PornoramaIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?pornorama\.com/video(?P<id>[0-9]+)(?:.*)'
+    _TEST = {
+        'url': 'http://www.pornorama.com/video1416287/amy_my_neighbor_fucked',
```

Comment:
This looks like copyrighted content. youtube-dl does not support sites dedicated to [copyright infringement](https://github.com/rg3/youtube-dl#can-you-add-support-for-this-anime-video-site-or-site-which-shows-current-movies-for-free). Do they have rights to distribute this? If not provide [an example URL](https://github.com/rg3/youtube-dl/blob/master/README.md#is-the-description-of-the-issue-itself-sufficient) that does not violate any copyrights.